webapp/src/css/panel.less

```diff
@@ -13,13 +13,6 @@
       border-top: 1px solid @separator-color;
       clear: right;
     }
-    &.panel-open {
```

Comment:
Simplified css to simplified html structure with less unnecessary wrappers.
and fixed styling for collapsible group to match with master, because the new component of bootstrap for angular uses a bit different layout